showcase/app/styles/showcase-pages/application-state.scss

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// APPLICATION-STATE
+
+body.application-state {
+  .shw-component-application-state-button-highlight {
+    margin: 24px 0;
+  }
+
+  .shw-component-application-state-layout-highlight {
+    .shw-flex__item {
+      outline: 1px dashed #78909c;
+    }
+
+    .hds-application-state {
+      outline: 2px dotted magenta;
```

Comment:
Nit: Maybe add a comment that the named colors are used purposely for Showcase purposes. I tend to use color names like this myself when testing layouts so at first glance they jump out to me as perhaps being mistakes which could catch up other reviewers also.